previewers/betatest/js/rocrate.js

```diff
@@ -0,0 +1,21 @@
+$(document).ready(function () {
+    startPreview(true);
+});
+
+function translateBaseHtmlPage() {
+    var mdPreviewText = $.i18n("mdPreviewText");
```

Comment:
The `mdPreviewText` variable can be removed here and its value directly added to the `text()` function.